src/lib/registration/registration.rb

```diff
@@ -311,13 +308,10 @@ def service_for_product(product, &block)
     def update_services(product_service)
       old_service = product_service.obsoleted_service_name
       # sanity check
-      if old_service && !old_service.empty? && old_service != product_service.name
-        # old_service comes from SCC. So it could be that we have already removed
-        # this service from the system meanwhile --> checking first.
-        if ::Registration::SwMgmt.service_installed?(old_service)
-          log.info "Found obsoleted service: #{old_service}"
-          ::Registration::SwMgmt.remove_service(old_service)
-        end
+      if old_service && !old_service.empty? && old_service != product_service.name &&
```

Comment:
```suggestion
      # old_service comes from SCC. So it could be that we have already removed
      # this service from the system meanwhile --> checking first.
      if old_service && !old_service.empty? && old_service != product_service.name &&
```